### PR TITLE
[Ready]Silly Tips

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
100% needed, We should also update and add to CAS as well as other memey text strings
Side note: Drinking and coding is bad and leads to you not making a new branch. Leading to your master branch being messy!
[why]
Why not?

### CODE REVIEW

File: strings/sillytips.txt [hunk 1]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
 There are at least 11 ways to get through plastic flaps. How many can you name?
 FEED ME A STRAY CAT
 Did you know that tossing the clown into a singularity will either increase or decrease its size by a large amount?
-Don't worry, you can always circlejerk your hate on the discord about furries.
\ No newline at end of file
+Don't worry, you can always circlejerk your hate on the discord about furries.
+Play a Vampire, there over powerd and broken with no downsides that matter!
```
Comment (deathride58) [inline strings/sillytips.txt]: s/there over powerd/they're overpowered

File: strings/sillytips.txt [hunk 2]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+You can make a Beepsky with E-swords that will murder people for not wareing there IDs!
```
Comment (deathride58) [inline strings/sillytips.txt]: s/wareing/wearing
s/there/their

File: strings/sillytips.txt [hunk 3]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+You can make a Beepsky with Toy-E-Swords if you want. There like a cheaper Beepsky that will trip people and maybe shout on the sec channle about people its "Arresting"
```
Comment (deathride58) [inline strings/sillytips.txt]: s/There/It's
s/chanle/channel

File: strings/sillytips.txt [hunk 4]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+There is a magic chem that truns people into Cat Girls/Boys. Its called "Felinid Mutation Toxin"!
+Having Neko ears and tails or Cat ears and tails does not make you into a Felinid race!
+Mhelp is basically useless, and never used correctly. Being a Mentor just means that admins will do Mhelps for you and you get to learn the round type when someone asks you a basic question regarding there antag status.
+Toxins Bombs are not the only way to get Rnd points. Tho no one will ever do any other method to get sci points...
```
Comment (deathride58) [inline strings/sillytips.txt]: s/Rnd/RnD
s/Tho no one/However, nobody

File: strings/sillytips.txt [hunk 5]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+Plasma men are a powerful race with many perks! No really I sware! So what if they need a "ERP Plasma room" to fuck, who cares!
```
Comment (deathride58) [inline strings/sillytips.txt]: s/No really I sware!/No really, I swear!
s/if they need a/if they need an

File: strings/sillytips.txt [hunk 6]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+As a Cargo Tech make sure to allways buy a tesla to sell back to CC. They love those those. Trust me!
```
Comment (deathride58) [inline strings/sillytips.txt]: s/allways/always

File: strings/sillytips.txt [hunk 7]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+Help. 
+Maints.
+BZ stops or slows down Lings chem reaeration drastically, make sure to BZ flood the station when lings are confirmed!
```
Comment (deathride58) [inline strings/sillytips.txt]: s/reaeration/regeneration

**Review by deathride58 — DISMISSED** [review]

File: strings/sillytips.txt [hunk 8]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
 Did you know that tossing the clown into a singularity will either increase or decrease its size by a large amount?
-Don't worry, you can always circlejerk your hate on the discord about furries.
\ No newline at end of file
+Don't worry, you can always circlejerk your hate on the discord about furries.
+Play a Vampire, they're overpowered and broken with no downsides that matter!
+You can make a Beepsky with E-swords that will murder people for not wearing their IDs!
+You can make a Beepsky with Toy-E-Swords if you want. It's like a cheaper Beepsky that will trip people and maybe shout on the sec channel about people its "Arresting"
+There is a magic chem that truns people into Cat Girls/Boys. Its called "Felinid Mutation Toxin"!
+Having Neko ears and tails or Cat ears and tails does not make you into a Felinid race!
+Mhelp is basically useless, and never used correctly. Being a Mentor just means that admins will do Mhelps for you and you get to learn the round type when someone asks you a basic question regarding there antag status.
```
Comment (cebutris) [inline strings/sillytips.txt]:
```suggestion
Mhelp is basically useless, and never used correctly. Being a Mentor just means that admins will do Mhelps for you and you get to learn the round type when someone asks you a basic question regarding their antag status.
```

File: strings/sillytips.txt [hunk 9]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
...
+Toxins Bombs are not the only way to get RnD points. However, nobody one will ever do any other method to get sci points...
```
Comment (cebutris) [inline strings/sillytips.txt]:
```suggestion
Toxins Bombs are not the only way to get RnD points. However, nobody will ever do any other method to get sci points...
```
```suggestion
Toxins Bombs are not the only way to get RnD points. However, no-one will ever do any other method to get sci points...
```
take your pick

**Review by nik707 — DISMISSED** [review]
This looks ready to go.

**Review by nik707 — CHANGES_REQUESTED** [review]
Just fucking kidding

File: strings/sillytips.txt [hunk 10]
```diff
@@ -31,4 +31,16 @@ As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curat
 There are at least 11 ways to get through plastic flaps. How many can you name?
 FEED ME A STRAY CAT
 Did you know that tossing the clown into a singularity will either increase or decrease its size by a large amount?
-Don't worry, you can always circlejerk your hate on the discord about furries.
\ No newline at end of file
+Don't worry, you can always circlejerk your hate on the discord about furries.
+Play a Vampire, they're overpowered and broken with no downsides that matter!
+You can make a Beepsky with E-swords that will murder people for not wearing their IDs!
+You can make a Beepsky with Toy-E-Swords if you want. It's like a cheaper Beepsky that will trip people and maybe shout on the sec channel about people its "Arresting"
+There is a magic chem that truns people into Cat Girls/Boys. Its called "Felinid Mutation Toxin"!
```
Comment (nik707) [inline strings/sillytips.txt]: truns > turns

**Review by nik707 — CHANGES_REQUESTED** [review]

File: strings/sillytips.txt (hunk 10 above)
Comment (nik707) [reply strings/sillytips.txt]: Fix this and I'll merge it.

**Review by nik707 — APPROVED** [review]

**Pull request merged**